models/forms/CalendarEntryForm.php

```diff
@@ -68,6 +68,11 @@ class CalendarEntryForm extends Model
      */
     public $timeZone;
 
+    /**
+     * @var string location in calendar form
+     */
+    public $location;
```

Comment:
Is there a reason we do not use the `$entry->location` here? The form usually only holds additional fields, which may need further validation/translation. Other fields as `title` for example are directly loaded into the entry since they doe not require special treatment.

Reply:
Sorry, my error, fixed in commit https://github.com/humhub/humhub-modules-calendar/commit/7ac1d99f02bdca026f96a4c5fac801e191e4482a